package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@coinbase/wallet-sdk",
-  "version": "3.0.7",
+  "version": "3.0.6",
```

Comment:
this is a downgrade; it probably shouldn't allow this

Reply:
afaik, 3.0.6/3.0.7 bumps were for testing and didn't have tagging to them. latest npm version is 3.0.5.
cleaned up master.

Reply:
Oh this version update happened during testing so a tag was never created and deployed